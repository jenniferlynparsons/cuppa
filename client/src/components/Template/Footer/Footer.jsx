import React from "react";

const Footer = () => (
  <footer className="footer">
    <div className="content has-text-centered is-size-7">
      <p>
        Created with love by{" "}
        <a href="http://jenniferlynparsons.net">Jennifer Lyn Parsons</a>
      </p>
    </div>
  </footer>
);

export default Footer;
