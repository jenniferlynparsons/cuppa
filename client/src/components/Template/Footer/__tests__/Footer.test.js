import React from "react";
import { create } from "react-test-renderer";
import Footer from "../";

test("snapshot", () => {
  const c = create(<Footer />);
  expect(c.toJSON()).toMatchSnapshot();
});
