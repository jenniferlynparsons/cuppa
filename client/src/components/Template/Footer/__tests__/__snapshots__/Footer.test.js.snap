// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<footer>
  <p>
    This is a Footer
  </p>
</footer>
`;
